example/otel-collector/main.go

```diff
@@ -80,8 +80,9 @@ func initProvider() func() {
 	pusher.Start()
 
 	return func() {
-		bsp.Shutdown() // shutdown the processor
-		handleErr(exp.Shutdown(context.Background()), "failed to stop exporter")
+		ctx := context.Background()
+		_ = bsp.Shutdown(ctx) // shutdown the processor
```

Comment:
Should we print this error?

Reply:
Added to error handler like on the next line